src/content/docs/new-relic-solutions/observability-maturity/operational-efficiency/dg-optimizing.mdx

```diff
@@ -73,7 +78,10 @@ Use the following examples to help you visualize how you would assess your own t
   >
     An account is ingesting about 20% more than they had budgeted for.  They have been asked by a manager to find some way to reduce consumption.  Their most important value driver is `Uptime, performance, and reliability`
 
-    ![Value Drivers Uptime](images/oma-oe-dg-value-driver-uptime.png)
+        <img
```

Comment:
@JimHagan, this image is pretty large compared to surrounding text. I'm not sure if this is an artifact of the image migration or whether this has always been large. This might call for some inline styling.

I see that similar images appear below. You might consider reducing those, too.

Reply:
@rhetoric101 It looks like those images are bigger with default HTML tags vs. markdown.  I'm going to update the width to 400.  It seems to work okay.